src/content/changes/documentation/2024-12-17.md

```diff
@@ -0,0 +1,13 @@
+---
+date: 2024-12-17T14:00:00
+title: Documentation Release v2.2.1379
+changes_categories:
+  - Documentation
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-docs
+---
+
+### Added
+- Use `requiredDuringSchedulingIgnoredDuringExecution` for `podAntiAffinity`
+- Add `PodDisruptionBudget` (PDB) for all deployments
+- Add `RollingUpdate` strategy with `maxSurge: 1` and `maxUnavailable: 1`
```

Comment:
Just delete this file and drop the changelog. It seems this repository is special and doesn't track changes in the form of a CHANGELOG.md.